src/app/navbar/navbar.component.scss

```diff
@@ -14,18 +14,34 @@
  *    limitations under the License.
  */
 
+@import 'materialColorScheme.scss';
+
 #dropdown-main {
   font-weight: bold;
   text-decoration: none;
   color: white;
 }
 
 .icon {
-  padding: 0 14px;
+  padding: 14px 14px;
   font-weight: bold;
   text-decoration: none;
   color: white;
   font-size: 14px;
+  border-radius: 0;
+  height: 64px;
+}
+
+.icon:hover {
+  background-color: #112246;
+}
+
+.icon-active {
+  background-color: mat-color($dockstore-app-primary, darker);
+}
+
+.menu-item-active {
+  background-color: #dfe0e2;
```

Comment:
ditto

Reply:
This color appears to be in the gray scale of kim's color palette, but the gray scale isn't in `materialColorScheme.scss`. The color is used in 3 styles in `styles.scss` and in the about component as well

Reply:
See if you can somehow specify it in the material color scheme and then reference it. if not possible via that, do something from https://sass-lang.com/documentation/variables